modules/installation-mirror-repository.adoc

```diff
@@ -108,7 +108,8 @@ $ RELEASE_NAME="okd"
 endif::[]
 
 ifndef::openshift-origin[]
-.. Export the type of architecture for your server, such as `x86_64`:
+.. Export the type of architecture for your server, such as `x86_64` 
```

Comment:
shouldn't `server` be `cluster nodes` here?

Reply:
I believe that `server` is right in the context of this procedure, but it is kinda confusing. I'll check with the writer as well for this. Thank you!

Reply:
@kelbrown20 I made a mistake, and sorry for troubling you again,  the stable version for arm is `quay.io/openshift-release-dev/ocp-release:4.11.0-fc.3-aarch64`, so `ARCHITECTURE` parameter need to change from arm64 to aarch64.